bluetooth/data_validation/directional_bias/matching_bt_and_here/create-view-validation_bluetooth.sql

```diff
@@ -0,0 +1,35 @@
+--Find out reference_id for bt sensor that was used in king_pilot (OLD, NOT USED)
```

Comment:
Can you put this in a separate pull request and give the view a meaningful name in the `bluetooth` schema?